chatcommands.py

```diff
@@ -583,17 +583,20 @@ def reject(msg, args, alias_used="reject"):
         raise CmdException("Please provide an adequate reason for rejection so the user"
                            " can learn from their mistakes. Use `-force` to force the reject")
     # Forward this, because checks are better placed in gitmanager.py
+    rejected_image="https://camo.githubusercontent.com/77d8d14b9016e415d36453f27ccbe06d47ef5ae2/68747470733a2f2f726173746"
+        "5722e736869656c64732e696f2f62616467652f626c61636b6c6973746572732d72656a65637465642d7265642e706e67"
     try:
         message_url = "https://chat.{}/transcript/{}?m={}".format(msg._client.host, msg.room.id, msg.id)
         chat_user_profile_link = "https://chat.{}/users/{}".format(
             msg._client.host, msg.owner.id)
-        comment = "[Rejected]({}) by [{}]({}) in {}. Reason provided: '{}'\n\n![Rejected with SmokeyReject]({})".format(
-            message_url, msg.owner.name, chat_user_profile_link, msg.room.name,
-            reason,
-            "https://camo.githubusercontent.com/77d8d14b9016e415d36453f27ccbe06d47ef5ae2/68747470733a2f2f7261737465722e"
-            "736869656c64732e696f2f62616467652f626c61636b6c6973746572732d72656a65637465642d7265642e706e67")
-        message = GitManager.reject_pull_request(pr_id, comment)
-        return message
+        if comment != '':
+            comment = "[Rejected]({}) by [{}]({}) in {}. Reason provided: '{}'\n\n![Rejected with SmokeyReject]({})"
+                .format(message_url, msg.owner.name, chat_user_profile_link, msg.room.name,reason,rejected_image)
+        else:
```

Comment:
This path appears to never actually run `reject_pull_request`. That's most likely an indenting issue in those lines below.

Reply:
there's other indenting issues in the code too makyen, as well as a "Variable Not Found" type error because comment isn't defined anywhere earlier, so it's a nonexistent variable at the tie of the `if` test.